plugins/Files/js/components/filetransfers.js

```diff
@@ -12,8 +12,8 @@ const FileTransfers = ({uploads, downloads, actions}) => {
 			<div className="close-button" onClick={onCloseClick}>
 				<i className="fa fa-times fa-2x"></i>
 			</div>
-			{uploads.size > 0 ? <UploadList uploads={uploads} /> : null}
 			{downloads.size > 0 ? <DownloadList downloads={downloads} onDownloadClick={onDownloadClick} /> : null}
+			{uploads.size > 0 ? <UploadList uploads={uploads} /> : null}
```

Comment:
no, `onUploadClick` gets passed to transfer list's `onTransferClick`, which has an es6 default parameter value.  If you pass undefined, it's the same as passing `() => {}`